controller/error.controller.js

```diff
@@ -0,0 +1,7 @@
+exports.handelPsqlError = (err, req, res, next) => {
```

Comment:
This shouldn't be needed yet. It looks good for future endpoints though 👍 
Make sure only essential code makes it on to Main.